app/src/main/java/com/android/solvit/shared/ui/booking/Booking.kt

```diff
@@ -303,88 +285,137 @@ fun ServiceBookingScreen(
+                                            text = time,
+                                            fontWeight = FontWeight.Bold,
+                                            fontSize = 15.sp,
+                                            textAlign = TextAlign.Center,
+                                            color = colorScheme.onPrimary)
+                                      }
+                                  if (acceptedOrScheduled && isSeeker) {
+                                    DateAndTimePickers(request!!, requestViewModel)
                                   }
                                 }
-                          }
+                              }
                         }
                   }
+              // Problem description section
+              Spacer(modifier = Modifier.height(16.dp))
+              Text(
+                  text = "Problem Description", // Title for the problem description
+                  fontSize = 20.sp,
+                  fontWeight = FontWeight.Bold,
+                  modifier = Modifier.padding(bottom = 8.dp).testTag("problem_description_label"))
+
+              Card(
+                  modifier = Modifier.fillMaxWidth(),
+                  shape = RoundedCornerShape(16.dp),
+                  colors = CardDefaults.cardColors(containerColor = colorScheme.background),
+                  border = BorderStroke(2.dp, colorScheme.onBackground.copy(alpha = 0.12f)),
+                  elevation = CardDefaults.cardElevation(0.dp) // No shadow
+                  ) {
+                    Column(modifier = Modifier.padding(8.dp)) {
+                      Text(
+                          text = request!!.description,
+                          fontSize = 16.sp,
+                          fontWeight = FontWeight.Bold,
+                          color = colorScheme.onSurface,
+                          modifier = Modifier.testTag("problem_description"))
+                    }
+                  }
+
+              if (!request!!.imageUrl.isNullOrEmpty()) {
+                Spacer(modifier = Modifier.height(16.dp))
+                Text(
+                    text = "Provided Image", // Title for the image display
+                    fontSize = 20.sp,
+                    fontWeight = FontWeight.Bold,
+                    modifier = Modifier.padding(bottom = 8.dp).testTag("problem_image_label"))
+
+                Card(
+                    modifier = Modifier.fillMaxWidth(),
+                    shape = RoundedCornerShape(16.dp),
+                    colors = CardDefaults.cardColors(containerColor = colorScheme.background),
+                    border = BorderStroke(1.dp, colorScheme.onSurface.copy(alpha = 0.12f)),
+                    elevation = CardDefaults.cardElevation(0.dp) // No shadow
+                    ) {
+                      AsyncImage(
+                          modifier = Modifier.fillMaxSize(),
+                          model = request!!.imageUrl,
```

Comment:
I feel like the image here looks massive with `.fillMaxSize()`, I would put the same height (around `200.dp`) as the map.

Reply:
Will do, thanks!